doc/nrf/known_issues.rst

```diff
@@ -2920,7 +2921,7 @@ NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks f
 Zephyr
 ******
 
-.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
```suggestion
.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

This issue is fixed and fix is https://github.com/nrfconnect/sdk-zephyr/commit/f4c6943be9e4824bea8ded30f5682143ca5df15e